lib/services/livesync/livesync-service.ts

```diff
@@ -3,11 +3,9 @@ import * as helpers from "../../common/helpers";
 import * as path from "path";
 import * as semver from "semver";
 import * as fiberBootstrap from "../../common/fiber-bootstrap";
-
-let gaze = require("gaze");
+let choki = require("chokidar");
```

Comment:
Also it's not added to package.json, so it will not be installed on any machine.